cmd/eksctl-anywhere/cmd/internal/commands/artifacts/noop.go

```diff
@@ -9,3 +9,5 @@ import (
 type Noop struct{}
 
 func (*Noop) Download(ctx context.Context, bundles *releasev1.Bundles) {}
+
+func (*Noop) Push(ctx context.Context, bundles *releasev1.Bundles) {}
```

Comment:
What's the purpose of this?

Reply:
FileImporter is only needed in case of curated packages hence noop provides a cleaner mechanism to provide a no operation for flows that don't need this.